config/initializers/rack_mini_profiler.rb

```diff
@@ -0,0 +1,5 @@
+Rack::MiniProfiler.config.position = 'left'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.